.chglog/CHANGELOG.tpl.md

```diff
@@ -0,0 +1,56 @@
+{{ if .Versions -}}
+<a name="unreleased"></a>
+## [Unreleased]
+
+{{ if .Unreleased.CommitGroups -}}
+{{ range .Unreleased.CommitGroups -}}
+### {{ .Title }}
+{{ range .Commits -}}
+- {{ if .Scope }}**{{ .Scope }}:** {{ end }}{{ .Subject }}
+{{ end }}
+{{ end -}}
+{{ end -}}
+{{ end -}}
+
+{{ range .Versions }}
+<a name="{{ .Tag.Name }}"></a>
+## {{ if .Tag.Previous }}[{{ .Tag.Name }}]{{ else }}{{ .Tag.Name }}{{ end }} - {{ datetime "2006-01-02" .Tag.Date }}
+{{ range .CommitGroups -}}
+### {{ .Title }}
+{{ range .Commits -}}
+- {{ if .Scope }}**{{ .Scope }}:** {{ end }}{{ .Subject }}
+{{ end }}
+{{ end -}}
+
+{{/* tags before 0.8.0 have no grouping */}}
+{{ if regexMatch "^0.[12345678].0$" .Tag.Name }}
+{{ range .Commits -}}
+- {{ .Header }}
+{{ end }}
+{{ end }}
```

Comment:
This template was unchanged, except this block. By default, when there is no type (type is the `fix` or `chore` or `feat` prefix), the tool skips commits. Since we don't have any types in commit messages prior this PR, we would have pretty much empty CHANGELOG up until 0.8.0 (this commit). This fixes it.